nltk/tag/stanford.py

```diff
@@ -133,6 +133,11 @@ class StanfordPOSTagger(StanfordTagger):
     _JAR = 'stanford-postagger.jar'
 
     def __init__(self, *args, **kwargs):
+        # Standford POSTagger uses '#' as separator for Chinese texts.
+        if ('chinese-distsim.tagger' in args[0] or
+            'chinese-nodistsim.tagger' in args[0]) and \
+           'stanford-postagger.jar' in args[1]:
```

Comment:
Try to use explicit variable names rather that `args[0]`

This if statement can be simplified to:

```
if os.path.split(self.model_filename)[1].startswith('chinese-'):
    self._SEPARATOR = '#'
```